style.css

```diff
@@ -0,0 +1,402 @@
+@import url("https://fonts.googleapis.com/css2?family=Recursive&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Recursive", sans-serif;
+}
+
+body {
+  background-color: #f3f4f6;
+}
+
+.landing-page {
+  background-color: #f3f4f6;
+  text-align: center;
+  height: 200px;
+  width: 400px;
+  position: fixed;
+  top: 40%;
+  left: 50%;
+  margin-top: -100px;
+  margin-left: -200px;
+}
+
+.landing-page h3 {
+  margin-top: 30px;
+}
+
+.landing-page button {
+  margin-top: 30px;
+}
+
+.landing-page button a {
+  color: white;
+  font-family: "Recursive", sans-serif;
+  text-decoration: none;
+}
+
+.landing-page a {
+  color: black;
+  font-family: "Recursive", sans-serif;
+  text-decoration: none;
+}
+
+.landing-page > button a:hover {
+  color: white;
+}
+
+.copyright {
+  margin-top: 30px;
+}
+
+.nav {
+  grid-area: nav;
+  border-bottom: 1px solid black;
+  display: flex;
+  background-color: #eff6ff;
```

Comment:
use CSS variables for colors and spacing units.  helps to maintain consistency across website